cinema/migrations/0002_movie_image.py

```diff
@@ -14,6 +14,7 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='movie',
             name='image',
-            field=models.ImageField(blank=True, null=True, upload_to=cinema.models.Movie.movie_image_path),
+            field=models.ImageField(default=1, upload_to=cinema.models.upload_movie_image_path),

```

Comment:
what is that default=1?

Reply:
and don't delete migration, create a new one instead because some issues may be present in the future